easybuild/framework/easyconfig/format/pyheaderconfigobj.py

```diff
@@ -184,8 +186,12 @@ def parse_pyheader(self, pyheader):
 
         try:
             exec(pyheader, global_vars, local_vars)
-        except SyntaxError, err:
-            raise EasyBuildError("SyntaxError in easyconfig pyheader %s: %s", pyheader, err)
+        except Exception as err:
```

Comment:
doesn't pep8/pylint complain about this? Add an exception for this exception line 😉

Reply:
It probably does, but we're not consistently adding pylint `disable` comments anywhere

I agree it makes sense to do so, but then we need to: i) do it consistently, ii) let Travis complain when `pylint` doesn't agree on something, otherwise it's not worth the effort?

Reply:
So, what reason do we have for not adding it from now on? We need to start somewhere...